src/commands/toolchain.rs

```diff
@@ -42,6 +50,20 @@ pub struct UninstallCommand {
 #[derive(Debug, Parser)]
 pub struct ListRevisionsCommand {}
 
+#[derive(Debug, Deserialize, Parser)]
+pub struct ExportCommand {
+    /// Toolchain to export, [possible values: latest, beta-1, beta-2, beta-3, beta-4, nightly].
+    /// The default toolchain will be exported if name isn't specified
+    pub name: Option<String>,
```

Comment:
Why does this not allow custom toolchains?

Reply:
> Why does this not allow custom toolchains?

Nop, there is no limit for name to be non-custom toolchains, but the channel does